New version: MarekJasinski.FreeCommanderXE version 2024.0.0.901

aisgbnok opened the pull request:
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Is there a linked Issue?
  - Fixes #167900

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/167958)

Dragon1573 commented:
Manually validated. I can't reproduce https://github.com/microsoft/winget-pkgs/labels/Validation-Unattended-Failed

```text
23:07:41 C:\...\winget-pkgs  [pull/167958 +0 ~1 -0 !] 1ms pwsh> .\Tools\SandboxTest.ps1 .\manifests\m\MarekJasinski\FreeCommanderXE\2024.0.0.901\
--> Validating Manifest
清单验证成功。

--> Checking dependencies
--> Starting Windows Sandbox, and:
    - Mounting the following directories:
      - C:\Users\Dragon1573\AppData\Local\Temp\SandboxTest as read-only
      - C:\Repository\winget-pkgs as read-and-write
    - Installing WinGet
    - Configuring Winget
    - Installing the Manifest 2024.0.0.901
    - Refreshing environment variables
    - Comparing ARP Entries
```

```text
--> Installing WinGet
--> Disabling safety warning when running installer
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Enabled admin setting 'LocalManifestFiles'.
Enabled admin setting 'LocalArchiveMalwareScanOverride'.

--> Installing the Manifest 2024.0.0.901

Found FreeCommander XE [MarekJasinski.FreeCommanderXE] Version 2024.0.0.901
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://freecommander.com/downloads/FreeCommanderXE-32-public_setup.zip
  ██████████████████████████████  14.5 MB / 14.5 MB
Successfully verified installer hash
Extracting archive...
Successfully extracted archive
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName                       DisplayVersion Publisher      ProductCode                                Scope
-----------                       -------------- ---------      -----------                                -----
FreeCommander XE Build 901 32-bit 2024.0.0.901   Marek Jasinski {D3C705DC-9743-4FEF-8358-E1AC9FA69C73}_is1 Machine
```

![image](https://github.com/user-attachments/assets/fbfc3854-4a1f-47bd-be8b-16cf8a6dd641)

stephengillie commented:
Automatic Validation ended with:


(Automated response - build 887.)

aisgbnok committed:
New version: MarekJasinski.FreeCommanderXE version 2024.0.0.901

aisgbnok commented:
> @aisgbnok & @Dragon1573 - is another installer switch needed to clear this message?

Yeah, as part of redoing this package/manifest, because it hasn't been touched in a while, I removed the `InstallerSwitches`. Winget is usually pretty good at passing the needed ones automatically based on `InstallerType`. During manual validation in a VM (W11 build 26120.1350) the install prompt never appeared. However, based on the results of this validation pipeline things may be different in Windows 10 and older builds. 

Therefore, I've added back:

```
InstallerSwitches:
  Silent: /SP- /VERYSILENT /NORESTART
  SilentWithProgress: /SP- /SILENT /NORESTART
```

It should pass appropriately now. Thanks.
